packages/framework/src/Resources/config/services.yml

```diff
@@ -292,6 +292,9 @@ services:
 
   Shopsys\FrameworkBundle\Component\Setting\Setting: ~
 
+  Shopsys\FrameworkBundle\Component\Setting\ClearSettingsCacheDoctrineSubscriber:
+    tags: ['doctrine.event_subscriber']
```

Comment:
This could be handled by `autoconfigure`, right?

Reply:
Yes, it could be, but I didn't feel the need to set it up for autoconfiguration because of a single service (other services are either listeners, or the have specific priority set).